lib/evmone/eof.cpp

```diff
@@ -514,11 +519,13 @@ std::variant<EOF1Header, EOFValidationError> validate_eof1(
         offset += code_size;
     }
 
+    std::set<uint16_t> accessed_code_sections = {0};
```

Comment:
You probably should use `std::unordered_set` as we don't care if the inserted values are in order.

Reply:
It's using `unordered_set` now.